dGame/dMission/RacingTaskParam.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <cstdint>
+
+enum class RacingTaskParam : int64_t {
```

Comment:
This enum shouldn't be its own file, should be in one of the racing components imo

Reply:
or dCommonVars

Reply:
I believe putting it into its own file is fine, would prefer that rather than clod up dCommonVars even more

Reply:
Whatever you do, do not put it in dCommonVars.

Reply:
Lol, I just thought that was where all enums were slapped

Reply:
Since MissionTaskType is in its own file I think this fits well in its own file because these values are only used for racing.  Have updated the enum to be a int32_t as I recall that enums shouldnt be of size 64.